api/graphql/schema.graphql

```diff
@@ -273,6 +278,8 @@ type Album {
   path: [Album!]!
 
   shares: [ShareToken!]!
+
+  coverID: Int!
```

Comment:
Isn't this redundant, as it can be derived from the `thumbnail: Media` attribute?

Reply:
I did initially think the same, but graphql throws up several errors if it can't find cover_id in the schema - left it in for now, will try and figure out why later.